src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -177,14 +204,22 @@ const chatReportSelector = (report) =>
  */
 const reportActionsSelector = (reportActions) =>
     reportActions &&
-    _.map(reportActions, (reportAction) => ({
-        errors: lodashGet(reportAction, 'errors', []),
-        message: [
-            {
-                moderationDecision: {decision: lodashGet(reportAction, 'message[0].moderationDecision.decision')},
-            },
-        ],
-    }));
+    _.map(reportActions, (reportAction) => {
+        const {reportActionID, parentReportActionID, actionName, errors = []} = reportAction;
+        const decision = _.get(reportAction, 'message[0].moderationDecision.decision');
+
+        return {
+            reportActionID,
+            parentReportActionID,
+            actionName,
```

Comment:
Can you add comment why these are added?

Reply:
We need them to get the parentReportAction, because the `ReportUtils.getParentReportAction()` method is deprecated.

Reply:
Add a comment in the code explaining why we need them